src/components/calculator.js

```diff
@@ -0,0 +1,10 @@
+function Calc() {
+  return (
+   <div className="calculator">
```

Comment:
- [ ] Please the calculator component should be a class-based component and it should look something like this.
![image](https://user-images.githubusercontent.com/74744787/185773473-48d081f8-0c9e-4641-9401-c1215ad83f0b.png)